src/main/java/org/team4909/powerup2018/Robot.java

```diff
@@ -178,6 +177,8 @@ public void robotPeriodic(){
     public void teleopPeriodic() {
         //System.out.println(drivetrain.getHeading());
 
+        arduino.send(23);
```

Comment:
This might be my code, but you should get rid of this, b/c then it'll override it in teleop.